ui/src/shared/components/TimeMachine/TimeMachine.tsx

```diff
@@ -536,6 +528,16 @@ class TimeMachine extends PureComponent<Props, State> {
     return isInCEO ? QueryUpdateState.CEO : QueryUpdateState.DE
   }
 
+  private get queriesForVis(): Query[] {
+    const {script, timeRange, queryDrafts} = this.props
+    const id = _.get(queryDrafts, 'id', '')
+    if (this.isFluxSource) {
+      return [{text: script, id, queryConfig: null}]
```

Comment:
what do you think about clarifying what is being returned here by assigning this to a named object before returning it?

Reply:
sure